docs/notebooks/usage.ipynb

```diff
@@ -8,9 +8,7 @@
     "\n",
```

Comment:
<p>Maybe make the sentence "Indicators should not be confused with <em>indices</em>, which define the algorithmic layer of each indicator." in bold? </p><p>Also, it might help the reader if this paragraph is split up into bullets/fragments. I know that for me, it's hard to parse what's the relevant information when it's so large.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/1472/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done